packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -98,6 +99,44 @@ export interface IMockContainerRuntimePendingMessage {
 	localOpMetadata: unknown;
 }
 
+/**
+ * Options for the container runtime mock.
+ */
+export interface MockContainerRuntimeOptions {
+	/**
+	 * Configures the flush mode for the runtime. In Immediate flush mode the runtime will immediately
+	 * send all operations to the driver layer, while in TurnBased the operations will be buffered
+	 * and then sent them as a single batch when `flush()` is called on the runtime.
+	 *
+	 * By default, flush mode is Immediate.
```

Comment:
Why is the default flush mode Immediate here? Compared to the real runtime where its TurnBased?

Reply:
All current tests are de facto using flushmode immediate. I will experiment with the other flushmode and see what happens (maybe we'll catch other bugs), but I don't want to change the defaults with this PR.